x/payment/types/builder.go

```diff
@@ -64,17 +65,41 @@ func (k *KeyringSigner) QueryAccountNumber(ctx context.Context, conn *grpc.Clien
 	return nil
 }
 
+func (k *KeyringSigner) UpdateAccountFromClient(clientCtx client.Context) error {
+	// clientCtx.AccountRetriever.
```

Comment:
signaling in case you forgot to  delete this comment :D

Reply:
thanks!